### ISSUE TITLE
Example Dataset Error

### ISSUE THREAD

**Issue by AmmarDharani** [ajitjohnson/arseq#8]
When using the sample dataset given on your website and inherent to the package, an error is thrown: 
Error in which(meta == groups[i], arr.ind = T)[1, ] : 
  subscript out of bounds

How can I get rid of this error and see the results?

**Comment by ajitjohnson** [comment]
Hi,

Can you provide a bit more detail on the exact set of scripts that you ran? Thanks.

**Comment by AmmarDharani** [comment]
Yes, this was the script I used (attached image), which is the same one you posted on your website. Thanks!
<img width="915" alt="Screen Shot 2020-06-11 at 2 37 56 PM" src="https://user-images.githubusercontent.com/37277300/84431654-42898c00-abf1-11ea-939b-09ab2b6611c1.png">

**Comment by ajitjohnson** [comment]
Ah, I see where the problem is. Looks like I had not updated the blog post. I had changed `treatment1` to `drug_A` in the example files.

Use the following instead
```
data("example_data")
data("example_meta")
arseq (data = example_data, meta = example_meta, design = "treatment", contrast = list(A = c("control"), B= c("drug_A")))
```
